File: Code coach/Hard/Password validation/password_validation.rb
password = gets.chomp
regex = /(?=.*[A-Za-z])(?=(.*\d){2,})(?=(.*[@$!%*#?&]){2,})[A-Za-z\d@$!%*#?&]{8,}/

if password.match?(regex)
	puts "Strong"
else
	puts "Weak"
end

File: Code coach/Hard/Password validation/password_validation.py
import re

password = input()
regex = r"(?=.*[A-Za-z])(?=(.*\d){2,})(?=(.*[@$!%*#?&]){2,})[A-Za-z\d@$!%*#?&]{8,}"

if re.match(regex, password):
  print("Strong")
else:
  print("Weak")
